Content.Server/Tabletop/TabletopCheckerSetup.cs

```diff
@@ -20,48 +17,64 @@ public override void SetupTabletop(TabletopSession session, IEntityManager entit
             SpawnPieces(session, entityManager, session.Position.Offset(-4.5f, 3.5f));
         }
 
-        private void SpawnPieces(TabletopSession session, IEntityManager entityManager, MapCoordinates topLeft, float separation = 1f)
+        private static void SpawnPieces(TabletopSession session, IEntityManager entityManager, MapCoordinates topLeft)
         {
             var (mapId, x, y) = topLeft;
+            var pieces = new EntityUid[42];
```

Comment:
Can just use a stackalloc Span<EntityUid> instead for stack arrays, at least on server.